src/lib.rs

```diff
@@ -159,6 +159,15 @@ fn model(state: &Mutex<AppState>, audio: AudioManager) {
 
     let mut last_update = Instant::now();
 
+    // In units of milliseconds. The time after the last dial alarm was acknowledged until
+    // the "Trial Complete!" splash screen is shown.
+    const SPLASH_SCREEN_DELAY: u64 = 10_000;
```

Comment:
nit: Use duration here to keep the unit with the type information
```suggestion
    const SPLASH_SCREEN_DELAY: Duration = Duration::from_secs(10);
```

Reply:
Good point

Reply:
Fixed